addons/mail/static/src/web/chatter.xml

```diff
@@ -74,10 +74,8 @@
         </div>
         <t t-if="state.thread.composer.type">
             <t t-if="state.thread.composer.type === 'message'">
-                <div class="flex-shrink-0 px-3 pb-1 pt-2 text-truncate" style="margin-left:48px;">
-                    <em class="text-muted">
-                        To followers of:
-                    </em>
+                <small class="flex-shrink-0 px-3 pt-3 text-truncate" style="margin-left:48px;">
+                    <b>To:</b> <em class="text-muted"> Followers of </em>
```

Comment:
As before, for purely stylistic purpose, prefer to use Bootstrap classes such as `.fw-bold` or `.fst-italic`.